### ISSUE TITLE
How to measure NER model's loss in a validation set?

### ISSUE THREAD

**Issue by echatzikyriakidis** [explosion/spaCy#6574]
How to measure NER model's loss in a validation set?

I have it from training set but I want also for validation set to plot train/val losses per epoch.

Can someone give an example?

Thank you!
